test/crypto.test.js

```diff
@@ -20,13 +20,11 @@ describe("crypto methods", () => {
     let notImplementedError = require("../src/notImplementedError");
     let cryptoLib = require("../src/crypto");
     let crypto = new cryptoLib();
-  
 
     let methodNames = [
         "digest",
         "hmac",
         "null_concat",
-        "verify",
```

Comment:
Why is verify removed?

Reply:
Because verify doesn't exist inside the crypto.js file any more. I moved it to RGPM. Since it deals more with the passwords and not the crypto, I didn't think it was too relevant in the crypto file.